randovania/generator/item_pool/pickup_creator.py

```diff
@@ -117,8 +121,8 @@ def create_dark_temple_key(key_number: int,
             game=resource_database.game_enum,
             name=echoes_items.DARK_TEMPLE_KEY_MODEL,
         ),
-        item_category=ItemCategory.TEMPLE_KEY,
-        broad_category=ItemCategory.KEY,
+        item_category=item_database.item_categories["temple_key"],
```

Comment:
It's a bit bizarre, but maybe the temple key categories shouldn't be in the json because none of the MajorItem uses it.

Of course, all this code for creating the keys is somewhat special.